core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -76,6 +74,7 @@ class SnappySession(@transient private val sc: SparkContext,
 
   self =>
 
+  SnappyContext.initConf = sc.conf
```

Comment:
Required for the cases when SnappyStreamingContext is created without creating SnappySession or SnappyContext. See AQPStreamingSuite.